.pkgmeta

```diff
@@ -0,0 +1,16 @@
+package-as: PvPAssistant
```

Comment:
this file is especially for the bigwigs packager? Or does anything else "look" at it?

Reply:
This is only for the packager.